addons/mass_mailing/static/src/js/unsubscribe.js

```diff
@@ -15,45 +16,47 @@ odoo.define('mass_mailing.unsubscribe', function (require) {
     if (!$('.o_unsubscribe_form').length) {
         return $.Deferred().reject("DOM doesn't contain '.o_unsubscribe_form'");
     }
-    if (email != '' && email != undefined){
-        ajax.jsonRpc('/mailing/blacklist/check', 'call', {'email': email, 'mailing_id': mailing_id, 'res_id': res_id, 'token': token})
-            .then(function (result) {
-                if (result == 'unauthorized'){
-                    $('#button_add_blacklist').hide();
-                    $('#button_remove_blacklist').hide();
-                }
-                else if (result == true) {
-                    $('#button_remove_blacklist').show();
-                    toggle_opt_out_section(false);
-                }
-                else if (result == false) {
-                    $('#button_add_blacklist').show();
-                    toggle_opt_out_section(true);
-                }
-                else {
+    session.load_translations().then(function () {
+        if (email != '' && email != undefined){
+            ajax.jsonRpc('/mailing/blacklist/check', 'call', {'email': email, 'mailing_id': mailing_id, 'res_id': res_id, 'token': token})
+                .then(function (result) {
+                    if (result == 'unauthorized'){
+                        $('#button_add_blacklist').hide();
+                        $('#button_remove_blacklist').hide();
+                    }
+                    else if (result == true) {
+                        $('#button_remove_blacklist').show();
+                        toggle_opt_out_section(false);
+                    }
+                    else if (result == false) {
+                        $('#button_add_blacklist').show();
+                        toggle_opt_out_section(true);
+                    }
+                    else {
+                        $('#subscription_info').html(_t('An error occured. Please try again later or contact us.'));
```

Comment:
this one can be replaced by a `.text(...)`

Reply:
@mart-e Thanks for the explanations.
I've replaced the unnecessary `.html()`